### ISSUE TITLE
Generate subscription member-specific usage reports

### ISSUE THREAD

**Issue by joedeville** [COPIM/open-book-collective#30]
As a librarian or institutional decision-maker visiting an open access initiative’s profile page ...

...I want (COUNTER compliant) usage reports for members of my institution ...

... so that I can assess ebook usage and compare with other ebooks in my collection, and include in print copy purchase decision making.

**Comment by joedeville** [comment]
Do we actually want to do this, given the issues around such data?

**Comment by MartinPaulEve** [comment]
This is nearly impossible. We have worked extensively on COUNTER implementation in Janeway and it is not worth the effort. It doesn't handle OA well and introduces loads of tracking overhead that is at best unhelpful and at worst unethical.

**Comment by rupertgatti** [comment]
I agree with Martin. It may be possible to integrate some form of reporting through the HIRMEOS database at some point. But should not be a priority now.

**Comment by joedeville** [comment]
Thanks both!
